schema/rpc.fbs

```diff
@@ -146,6 +146,13 @@ table OSCTrackersSetting {
 table TapDetectionSettings {
     tap_reset_delay: float = null;
     tap_reset_enabled: bool = null;
+    tap_reset_taps: uint16 = null;
+    tap_quick_reset_delay: float = null;
+    tap_quick_reset_enabled: bool = null;
+    tap_quick_reset_taps: uint16 = null;
```

Comment:
I think that a uint8 is probably sufficient, right?

Reply:
Yeah, I was just matching it up with what was in java if it can be smaller it should be